setup.py

```diff
@@ -12,7 +12,7 @@
     install_requires=[
         "celery==3.1.26post2",
         "coreapi==2.3.3",
-        "Django==2.1",
+        "Django==2.1.2",
```

Comment:
I think we might want to go with `"Django>=2.1.2,<2.2"`

Reply:
@nathanbegbie I've updated it, failed at the rebase thing though.